sdk/synapse/azure-analytics-synapse-accesscontrol/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2021-02-09)
 
+Regenerated with AutoRest & Swagger fixes.
```

Comment:
Customers may not know what regenerating from swagger means. So, it might be useful to call out a few prominent changes/fixes.

Reply:
Thanks! let me add more details